uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/implementation/addon/MqttMessageEvent.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.mqtt.client.control.implementation.addon;
+
+import com.aws.greengrass.testing.mqtt.client.Mqtt5Message;
+import com.aws.greengrass.testing.mqtt.client.control.api.ConnectionControl;
+import com.aws.greengrass.testing.mqtt.client.control.api.addon.EventFilter;
+import com.google.protobuf.ByteString;
+import lombok.NonNull;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+
+/**
+ * Implements received MQTT message event.
+ */
+public class MqttMessageEvent extends EventImpl {
+
+    private final ConnectionControl connectionControl;
+    private final Mqtt5Message message;
```

Comment:
add a getter for the message so we can inspect it outside of the event if we want.

Reply:
Added in be1c3a7d3605ffadf4a972c3b5c0b21ac0669512